### DISCUSSION TITLE
希望倍速的快捷键直接对应相应的倍速！

### DISCUSSION THREAD

**Discussion by LeahLixue** [the1812/Bilibili-Evolved#4446] [功能建议 / Ideas]
希望倍速的快捷键直接对应相应的倍速。
比如：
1倍速 Z, 
1.5倍速 X，
2倍速C，
3倍速V.
很多时候是直接想从1.5倍速转换到3倍速的！
望采纳！

**Comment by MC-dusk** [comment]
有B站原生的shift+1或者shift+2切换1、2倍速
